packages/rum/test/rum.spec.ts

```diff
@@ -519,11 +524,12 @@ describe('rum global context', () => {
     server = sinon.fakeServer.create()
     lifeCycle = new LifeCycle()
     RUM = startRum('appId', lifeCycle, configuration as Configuration, session, internalMonitoring) as RumApi
-    startViewCollection(location, lifeCycle, session)
+    viewCollection = startViewCollection(location, lifeCycle, session)
     server.requests = []
   })
 
   afterEach(() => {
+    viewCollection.stop()
```

Comment:
Same here, viewCollection may not be defined

Reply:
it is defined in the beforeEach, when can it not be defined?